### PR TITLE
sql: add `FLUSH PRIVILEGES` hint

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs/pull/835
@lilin90 @lonng PTAL when https://github.com/pingcap/docs/pull/835 is merged.

### CODE REVIEW

File: sql/user-account-management.md [hunk 1]
```diff
@@ -90,3 +90,13 @@ SET PASSWORD FOR 'root'@'%' = 'xxx';
 ```sql
 ALTER USER 'jeffrey'@'localhost' IDENTIFIED BY 'mypass';
 ```
+
+## `FLUSH PRIVILEGES` 
+
+如果授权表已被直接修改，运行如下命令立刻应用改动信息：
```
Comment (lilin90) [inline sql/user-account-management.md]:
```suggestion
如果授权表已被直接修改，运行如下命令可使改动立即生效：
```

Comment (lilin90) [reply sql/user-account-management.md]: Perhaps it is clearer this way :)

Comment (lonng) [reply sql/user-account-management.md]: Agreed

**Comment by lonng** [comment]
Rest LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Comment by CaitinChen** [comment]
@lilin90 PTAL again

**Pull request merged**